features/admins/admin_reorder_faq_questions.feature

```diff
@@ -0,0 +1,48 @@
+@admin
+Feature: Admin Actions to re-order FAQs
+  As an an admin
+  I want to be able to re-order the FAQs
+
+Scenario: Re-order the FAQs
```

Comment:
Just a nitpick and a suggestion! 

The suggestion: This could be shortened up a bit so it just checks the bits that are relevant to reordering FAQs -- we have a step to create several FAQs in one go, as well as a path definition that should go right to the FAQ reorder page.

The nitpick: It doesn't really _affect_ anything in the grand scheme of things, but in Cucumbers, `Given` is usually for set up steps (like `Given I am logged in as an admin`), `When` is for steps that take an action, and `Then` is for steps that check result of the action you took in the `When`s, so we might have something like:

```
Given I am logged in as an admin
  And 3 Archive FAQs exist
When I go to the FAQ reorder page
Then I should see "1. Number 1 Question" 
  And I should see etc
  And I should see etc
When I fill in etc
  And I fill in etc
  And I press "Update Positions"
  And I follow etc
Then etc
```